docs/source_es/quicktour.mdx

```diff
@@ -201,40 +212,32 @@ El tokenizador devolverá un diccionario conteniendo:
 
 Como con el [`pipeline`], el tokenizador aceptará una lista de inputs. Además, el tokenizador también puede rellenar (pad, en inglés) y truncar el texto para devolver un lote (batch, en inglés) de longitud uniforme:
 
-```py
->>> pt_batch = tokenizer(
-...     ["We are very happy to show you the 🤗 Transformers library.", "We hope you don't hate it."],
-...     padding=True,
-...     truncation=True,
-...     max_length=512,
-...     return_tensors="pt",
-... )
->>> # ===PT-TF-SPLIT===
->>> tf_batch = tokenizer(
-...     ["We are very happy to show you the 🤗 Transformers library.", "We hope you don't hate it."],
-...     padding=True,
-...     truncation=True,
-...     max_length=512,
-...     return_tensors="tf",
-... )
-```
+<frameworkcontent>
+  <pt>
+    ```py >>> pt_batch = tokenizer( ... ["We are very happy to show you the 🤗
+    Transformers library.", "We hope you don't hate it."], ... padding=True, ...
+    truncation=True, ... max_length=512, ... return_tensors="pt", ... ) ```
+  </pt>
+  <tf>
+    ```py >>> tf_batch = tokenizer( ... ["We are very happy to show you the 🤗
+    Transformers library.", "We hope you don't hate it."], ... padding=True, ...
+    truncation=True, ... max_length=512, ... return_tensors="tf", ... ) ```
+  </tf>
+</frameworkcontent>
```

Comment:
Please keep the original syntax here. The whole example is completely messed up.